ui/app/css/itcss/components/send.scss

```diff
@@ -70,4 +79,117 @@
   justify-content: space-between;
 }
 
-.send-screen-bolt-icon {}
\ No newline at end of file
+.send-screen-bolt-icon {}
+
+.selected-currency {
+  color: $curious-blue;
+}
+
+.unselected-currency {
+  cursor: pointer;
+}
+
+.send-screen-gas-input-customize {
+  color: $curious-blue;
+  font-size: 12px;
+  cursor: pointer;
+}
+
+.customize-gas-tooltip-container {
+  position: absolute;
+  left: 76px;
+  bottom: 50px;
+  width: 237px;
+  height: 307px;
+  background-color: white;
+  opacity: 1;
+  box-shadow: grey 0px 0px 5px;
+  z-index: 1050;
+  padding: 13px 19px;
+  font-size: 16px;
+  border-radius: 4px;
+  font-family: 'Avenir Next';
```

Comment:
is this font available in metamask?

Reply:
I've replaced it with a new font that I added "Lato" which is a near equivalent alternative.